packages/attestation-service/src/requestHandlers/attestation.ts

```diff
@@ -221,6 +228,30 @@ class AttestationRequestHandler {
 
     return attestation
   }
+
+  private async verifyPepperIfApplicable(): Promise<void> {
+    if (this.attestationRequest.phoneNumberSignature) {
```

Comment:
For backwards compatibility we need this to be optional, but we should make a plan to require this field in requests. I'd consider naming the function as if the pepper is always verified, and adding a TODO to remove the if statement once clients have upgraded. We should also add a zenhub ticket to track this